modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs

```diff
@@ -0,0 +1,63 @@
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<div id="flowChartView" style="display:none">
+    <button class="btn btn-primary" id="flowchartSaveBtn" onclick="saveFlowchart()">Save
+    </button>
+    <button class="btn btn-primary" id="saveAsPNGBtn">Download As PNG</button>
+    <br/><br/>
+
+    <div style="width:100%; height: 10%; white-space:nowrap;">
+                            <span style="display: inline-block; vertical-align: top; padding: 5px; width:15%">
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7